src/Hosting/Hosting/src/Internal/HostingEventSource.cs

```diff
@@ -72,6 +72,12 @@ public void UnhandledException()
         WriteEvent(5);
     }
 
+    [Event(6, Level = EventLevel.Informational)]
```

Comment:
Do you need to say no inlining?

cc @noahfalk

Reply:
I'm not sure why we have no inlining on the other methods...

Reply:
The NoInlining appears to be the result of hand-tuning one specific code path. I agree with @adityamandaleeka that this path isn't performance critical enough to merit overriding the JIT's default behavior (even if we were willing to hand tune, there is good chance we'd find the JIT's default inlining choice here is also the fastest choice)